File: Programmers/가장큰수.py
def solution(numbers):
    answer = ''
    numbers=list(map(str,numbers))
    data=sorted(numbers,key=lambda x:(x[0],x[1%len(x)],x[2%len(x)],x[3%len(x)]),reverse=True)
    answer=''.join(data)
    if int(answer)==0:
        answer='0'
    return answer

File: Programmers/가장큰수.js
function solution(numbers) {
  let str_numbers = numbers.map((i) => String(i));
  let answer = str_numbers
    .sort((a, b) => {
      return b + a - (a + b);
    })
    .join('');
  return answer[0] === '0' ? '0' : answer;
}
